slides/_naming_for_clarity.qmd

```diff
@@ -18,4 +18,53 @@ A few naming tips and conventions:
 
 ## Naming For Clarity
 
-Some examples:
\ No newline at end of file
+Some examples:
+
+- Pronounceable names without abbreviations: \
+ `ms`, `chclt`, `stm` --> `mass`, `chocolate`, `stem`
+- Naming for the content, not the type: \
+`array`, `list`, `age_int`, `country_set` --> `dogs`, `cats`, `age`, `countries`
+- Naming booleans: \
+`purple`, `not_plant`, `sidekick` --> `is_purple`, `is_plant`, `has_sidekick`
```

Comment:
When I render these the comparisons you want to get across are not super obvious.
Consider placing them in columns, or as a code block perhaps?

Reply:
Thanks, have updated accordingly :)